package.json

```diff
@@ -39,7 +39,7 @@
     "update-versions": "lerna version --yes --no-commit-hooks --no-push --exact --conventional-commits --no-git-tag-version",
     "commit": "git-cz",
     "coverage": "codecov || exit 0",
-    "add-cli-no-save": "yarn add @aws-amplify/cli-internal -W  && git checkout -- package.json",
+    "add-cli-no-save": "yarn add @aws-amplify/cli-internal@11.0.3 -W  && git checkout -- package.json",
```

Comment:
Should we change the CLI version in CCI config files?

Reply:
We need another change there. On it

Reply:
We should update the config.yml as well.